src/model/ModelController.h

```diff
@@ -9,8 +9,6 @@
 #include "persistence/Persistence.h"
```

Comment:
no need to include Persistence interface in ModelController interface?

Reply:
Fixed.